libdnf/dnf-context.cpp

```diff
@@ -229,6 +230,7 @@ dnf_context_init(DnfContext *context)
     priv->check_transaction = TRUE;
     priv->enable_filelists = TRUE;
     priv->zchunk = TRUE;
+    priv->write_history = FALSE;
```

Comment:
This should default to `TRUE`, no? Otherwise, we're regressing everywhere other than rpm-ostree.

Reply:
Yeah, we should be defaulting to `TRUE` here.

Reply:
ACK, changed to TRUE.